packages/_helpers/expand-transition.tsx

```diff
@@ -46,6 +46,7 @@ export function ExpandTransition({
       className={initialStyle}
       ref={expandableRef}
       aria-hidden={!show ? true : undefined}
+      data-testid="expand-transition"
```

Comment:
Maybe it was already discussed before, but I'm wondering if we should hardcode the data-testid string, or if it makes sense to have a `dataTestId` prop and maybe default it this way (or with a `w-` prefix?). Then both we and whoever uses our components should be able to rely on the `getByTestId()` even when multiple instances of our components are rendered. Here's an article about that: https://medium.com/@kavianrabbani/the-hidden-danger-of-hardcoded-data-testid-in-testing-8ec9e671955e

Reply:
Ideally our consumers shouldn't use our testid's for testing but if they do, we cannot stop them :)

Reply:
> Ideally our consumers shouldn't use our testid's for testing but if they do, we cannot stop them :)

Oh, I thought that us having `data-testid` in our components was supposed to be an added value for our users. But I guess we can always add a prop like `dataTestId` later if there's a well-grounded need for that 👍